agent/agent.go

```diff
@@ -3196,6 +3202,83 @@ func (a *Agent) loadProxies(conf *config.RuntimeConfig) error {
 	return persistenceErr
 }
 
+func (a *Agent) getPersistedTokens() (map[string]string, error) {
+	if !a.config.ACLEnableTokenPersistence {
+		return nil, nil
+	}
+
+	a.persistedTokensLock.RLock()
+	defer a.persistedTokensLock.RUnlock()
+
+	tokensFullPath := filepath.Join(a.config.DataDir, tokensPath)
+	tokensFile, err := os.Open(tokensFullPath)
```

Comment:
Why not `ioutil.ReadFile`?

Reply:
I just copy pasted from existing functions doing the same thing. I certainly could use `ioutil.ReadFile`